src/bp/ui-shared/src/typings.d.ts

```diff
@@ -133,7 +133,7 @@ declare module 'botpress/shared' {
 
   export const toast: {
     dismiss: (key: string) => void
-    success: (message: string | React.ReactElement, details?: string, options?: ToastOptions) => void
+    success: (message: string | React.ReactElement, details?: string, options?: ToyarastOptions) => void
```

Comment:
ToyarastOptions ?

Reply:
Sorry, that was meant to be Toastoptions.